ansible/lemmy.yml

```diff
@@ -17,6 +18,13 @@
   - name: install dependencies
     apt:
       pkg: ['nginx', 'docker-compose', 'docker.io', 'certbot', 'python-certbot-nginx']
+    when: not (ansible_distribution == 'Ubuntu' and ansible_distribution_major_version == '20')
```

Comment:
This is also going to be used for Ubuntu 21 or higher. Shouldnt it use a comparison with "less than" or "greated than"?

Reply:
What are the earliest versions you wish to support? At some point python2 support can just be dropped.

Reply:
Missed this comment. Not sure tbh, but we can keep it in for at least half a year more, until most people have upgraded their OS.